allensdk/brain_observatory/behavior/write_nwb/_schemas.py

```diff
@@ -29,7 +29,7 @@ class SessionData(RaisingSchema):
     movie_width = Int(required=True, description='width of field-of-view for 2p movie')
     container_id = Int(required=True, description='container that this experiment is in')
     sync_file = String(required=True, description='path to sync file')
-    segmentation_mask_image_file = String(required=True, description='path to segmentation_mask_image file')
```

Comment:
Looks like a bug

Reply:
We are passing both segmentation_mask_image_file and max_projection_file to the LIMS job, not substituting one for the other: http://stash.corp.alleninstitute.org/projects/TECH/repos/lims/browse/app/strategies/behavior_ophys_write_nwb_strategy.rb#65-66